From ee16ee20205ea374613c3ac7f50209451eb29068 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua.mayer97@gmail.com>
Date: Sun, 23 Apr 2017 18:02:40 +0200
Subject: [PATCH 1/4] clearfog: generate random MAC address

The Clearfog does not come with predetermined MACs.
Falling back to random ones ensures ethernet can still operate anyway.

Signed-off-by: Josua Mayer >josua.mayer97@gmail.com>
---
 configs/clearfog_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/clearfog_defconfig b/configs/clearfog_defconfig
index 1264871e83..41e94e6daf 100644
--- a/configs/clearfog_defconfig
+++ b/configs/clearfog_defconfig
@@ -47,6 +47,7 @@ CONFIG_DEBUG_UART=y
 CONFIG_DEBUG_UART_BASE=0xd0012000
 CONFIG_DEBUG_UART_CLOCK=250000000
 CONFIG_DEBUG_UART_SHIFT=2
+CONFIG_NET_RANDOM_ETHADDR=y
 CONFIG_SYS_NS16550=y
 CONFIG_USB=y
 CONFIG_DM_USB=y
-- 
2.12.2

